front/src/main.js

```diff
@@ -3,14 +3,17 @@ import App from "./App.vue";
 import router from "./router";
 import store from "./store";
 import vuetify from "./plugins/vuetify";
+import momentLocale from "moment/locale/ko";
 import KakaoMap from "./plugins/kakao-map";
 
 Vue.config.productionTip = false;
 Vue.use(KakaoMap);
+Vue.use(require("vue-moment"));
```

Comment:
```suggestion
const moment = require("moment");
require("moment/locale/ko");
Vue.use(require("vue-moment"), {
  moment,
});
```

이 방식으로 한다면 별도로 import 없이 plugin으로 프로젝트 내에서 전역적으로 사용할 수 있을거예요. 😄
한국어 패치까지 된 상태로요!
`momentLocale`의 의존성을 required로 추가해주었으니 import하고 Vue 안에 넣어줄 필요도 없어졌답니다!

Reply:
플러그인이라니 생각하지 못했던 방법이네요!
불필요한 코드도 사라지고 훨씬 깔끔해졌어요! 좋은 의견 감사합니다 :)